### ISSUE TITLE
Provide MailChimp with a custom From Email domain (but which domain?)

### ISSUE THREAD

**Issue by tfnico** [git/git.github.io#198]
MailChimp warns us about the following:

> Deliverability warning
> The From email address for this campaign is set as gitrevnews@gmail.com.
> In the near future, Gmail will implement authentication policies that could prevent your campaign from being delivered to Gmail addresses.
> 

Sounds like some spam prevention effort. The right thing for us to do would be to have something like feedback@gitrevnews.com - in any case, whatever From-email we specify, we'll have to verify the domain (I'm guessing we have to have a valid email there) with MailChimp.

Any ideas what domain we could use, @chriscool ? I only own tfnico.com and gitminutes.com. I could set up a forwarder on revnews@gitminutes.com to go to our gmail account, but it's still a bit weird.

**Comment by chriscool** [comment]
Yeah, I got the same warning, but I guessed that Nicola had ignored it so that it was safe to continue to ignore it.

I don't see a good solution, except perhaps asking the Git Project or the Conservancy to buy and keep a domain for us, or maybe lend us one they already have.
